pkg/importer/util.go

```diff
@@ -38,10 +40,20 @@ func MoveFile(src, dst string) error {
 	return err
 }
 
-// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file
-func StreamDataToFile(dataReader io.Reader, filePath string) error {
+// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file or block device
+func StreamDataToFile(dataReader io.Reader, filePath string, volumeMode v1.PersistentVolumeMode, scratchSpaceForBlockPV bool) error {
 	// Attempt to create the file with name filePath.  If it exists, fail.
-	outFile, err := os.OpenFile(filePath, os.O_CREATE|os.O_EXCL|os.O_WRONLY, os.ModePerm)
+	var outFile *os.File
+	var err error
+	if volumeMode == v1.PersistentVolumeBlock {
```

Comment:
If you take the comment from Copy into account, then you don't need the extra flag, since when we write to the scratch space the volume mode will always be file system, regardless of the target being block or not.